screens/TablesScreen.js

```diff
@@ -253,7 +252,7 @@ class TablesScreen extends React.Component {
                   >
                     <TouchableOpacity
                       onPress={() => {
-                        this.props.navigation.navigate('Tables')
+                        this.props.navigation.navigate('LoginSuccess')
```

Comment:
so after opening shift, why not just go to Tables?

Reply:
This is Cancel Button Before opening shift, so redirect to LoginSuccess, for ur above review, I will  be using errorAlert, successMessage every where it needs.

Reply:
Thank Shin.